lib/auth/auth_with_roles.go

```diff
@@ -1920,13 +1920,23 @@ func (a *ServerWithRoles) Ping(ctx context.Context) (proto.PingResponse, error)
 	if err != nil {
 		return proto.PingResponse{}, trace.Wrap(err)
 	}
-
+	heartbeat, err := a.authServer.GetLicenseCheckResult(ctx)
+	if err != nil {
+		return proto.PingResponse{}, trace.Wrap(err)
+	}
+	warnings := make([]string, 0)
```

Comment:
```suggestion
	var warnings []string
```

Make with an explicit len makes sense if you know ahead of time the exact length you want, but you don't. So I'd stick with this instead.